source/iface/usm_basic_concept.rst

```diff
@@ -2,288 +2,291 @@
+device, and the total size of all shared allocations
+in a context, are implementation-defined.
+
+Support for shared allocations on a specific device can be
+queried through ``sycl::aspect::usm_shared_allocations``.
+
+.. warning::
+
+  Not all devices may support concurrent access of a shared allocation with the host.
+
+If a device does not support this, host execution and device
+code must take turns accessing the allocation, so the host
+must not access a shared allocation while a kernel is executing.
+
+.. warning::
+
+  Host access to a shared allocation which is also accessed by an executing
+  kernel on a device that does not support concurrent access results in
+  undefined behavior.
+
+  If a device does support concurrent access, both the host and and
+  the device may atomically modify the same data inside an allocation.
+
+Allocations, or pieces of allocations, are now free to migrate to different
+devices in the same context that also support this capability.
+Additionally, many devices that support concurrent access may support
+a working set of shared allocations larger than device memory.
+
+Whether a device supports concurrent access with atomic modification of
+shared allocations can be queried through the aspect
+``sycl::aspect::usm_atomic_shared_allocations``.
+
+.. rubric:: Performance hints
+
+1. Performance hints for shared allocations may be specified by
+   the user by enqueueing ``prefetch`` operations on a device.
+   These operations inform the SYCL runtime that the specified
+   shared allocation is likely to be accessed on the device in
+   the future, and that it is free to migrate the allocation to
+   the device. If a device supports concurrent access to shared
+   allocations, then ``prefetch`` operations may be overlapped with
+   kernel execution. More about ``prefetch`` is found in
+   :ref:`sycl::queue shortcut functions <queue_shortcut>` and `Table 132`
+2. Users also may use the ``mem_advise`` member function to
+   annotate shared allocations with ``advice``. Valid ``advice`` is defined
+   by the device and its associated backend.
+   See :ref:`sycl::queue shortcut functions <queue_shortcut>`
+   and `Table 132` for more information.
+
+.. Future update: after updating "Expressing parallelism" add ref to Table 132
+
+.. rubric:: Example
+
+See `usm-example-1`_.
+
+
+System allocations
+------------------
+
+In the most capable systems, users do not need to use SYCL USM
+allocation functions to create shared allocations. The system
+allocator (``malloc``/``new``) may instead be used.
+Likewise, ``std::free`` and ``delete`` are used instead of ``sycl::free``.
+
+.. note::
+
+  Host and device allocations are unaffected by this change and
+  must still be allocated using their respective USM functions in order to
+  guarantee their behavior. Users may query the device to determine if
+  system allocations are supported for use on the device,
+  through ``sycl::aspect::usm_system_allocations``.
```

Comment:
I don't think the second sentence belongs in the note.  To match the description style of the earlier kinds of USM, please move the sentence about `sycl::aspect::usm_system_allocations` to before the note.

Reply:
I agree, it fits better before the note, rearranged in the latest commit.